sdk/keyvault/azure-security-keyvault-common/src/keyvault_pipeline.cpp

```diff
@@ -3,7 +3,7 @@
 
 #include <azure/core/http/http.hpp>
 
-#include "azure/keyvault/common/internal/keyvault_exception.hpp"
+#include "azure/core/exception.hpp"
```

Comment:
Move this to after the local and with <azure/core/exception.hpp>

We use "filename.hpp" for headers in the package and <filename.hpp> for includes from other packages.

Reply:
done